fairgraph/client.py

```diff
@@ -385,11 +385,14 @@ def _private_space(self):
         # temporary workaround
         return f"private-{self.user_info().identifiers[0]}"
 
-    def configure_space(self, space_name, types):
-        """
+    def configure_space(self, types, space_name=None):
```

Comment:
to avoid changing the order of arguments, I suggest giving `types` a default of None also.

also, the method should have a docstring